File: frontend/src/types/pessoa.ts
export interface Pessoa {
  id: number;
  nome: string;
  sexo: string;
  email: string;
  dataNascimento: string; // ISO string, ex: "2025-07-23T21:00:18.191Z"
  naturalidade: string;
  nacionalidade: string;
  cpf: string;
  endereco: Endereco;
}

export interface Endereco {
  logradouro: string;
  numero: string;
  complemento?: string;
  bairro: string;
  cidade: string;
  estado: string;
  cep: string;
}

File: backend/CadastroPessoasApi/Models/Pessoa.cs
﻿namespace CadastroPessoasApi.Models;

public class Pessoa
{
    public int Id { get; set; }
    public string Nome { get; set; } = null!;
    public string Email { get; set; }
    public string Sexo { get; set; }
    public DateTime DataNascimento { get; set; }
    public string Naturalidade { get; set; }
    public string Nacionalidade { get; set; }
    public string CPF { get; set; } = null!;
    public Endereco Endereco { get; set; }
    public DateTime DataCadastro { get; set; }
    public DateTime DataAtualizacao { get; set; }


}
